lte/gateway/c/oai/tasks/mme_app/mme_app_context.c

```diff
@@ -613,7 +613,7 @@ void mme_ue_context_update_coll_keys(
     h_rc = HASH_TABLE_KEY_NOT_EXISTS;
   }
 
-  if (HASH_TABLE_OK != h_rc) {
+  if ((HASH_TABLE_OK != h_rc) && (mme_teid_s11)) {
```

Comment:
@rsarwad , this change was added as part of https://github.com/magma/magma/pull/4141/files#diff-6c4967c552829a02382a628aec8b22501022d4c5b6b7609f95ef28ecd22e61fa , can you double check with @pruthvihebbani?